README.md

```diff
@@ -76,13 +76,13 @@ Load the template in your preamble:
 
 ### Package options
 
-| Option      | Description                                                       | Default |
-|-------------|-------------------------------------------------------------------|:-------:|
-| `bg-letter` | Loads a letter-sized background image                             |    ✓    |
-| `bg-a4`     | Loads an A4-sized background image                                |         |
-| `bg-print`  | Loads a printer-friendly background-image (only the footer image) |         |
-| `bg-full`   | Loads the full background image                                   |    ✓    |
-| `justified` | Justifies column copy                                             |         |
+| Option                   | Description                                                       | Default |
+|--------------------------|-------------------------------------------------------------------|:-------:|
+| `bg-a4` (deprecated)     | Loads an A4-sized background image                                |         |
+| `bg-full`                | Loads the full background image                                   |    ✓    |
+| `bg-letter` (deprecated) | Loads a letter-sized background image                             |         |
```

Comment:
Maybe just remove the note of the package options from the readme rather than marking them deprecated. The changelog says they are deprecated, and users will get a package warning if they try to use them. bg-print's description is a bit out of date. It now removes the background image rather than loading a printer friendly one. Also, bg-none is not in the list for some reason.

Reply:
Done.